### PR TITLE
code update

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
code update

### CODE REVIEW

File: stories/Molecules/Form/signup.scss [hunk 1]
```diff
@@ -40,12 +40,13 @@
 
   .form-submit {
     cursor: pointer;
-    font-family: $Font-Family-ProximaBold;
+    font-family: $Font-Family-ProximaNova;
```
Comment (surbhi90) [inline stories/Molecules/Form/signup.scss:43]: @harishkumar-srijan do we need to mention font family here

File: stories/Molecules/Sidebar-data/sidebardata.scss [hunk 2]
```diff
@@ -3,13 +3,14 @@
 @import '../../assets/scss/variables';
 
 .accordion-navbar__item--headertext {
+  @extend %img_none;
+
   border: none;
-  font-family: $Font-Family-ProximaBold;
   font-size: $font-size-16;
+  // font-family: $Font-Family-ProximaNova;
```
Comment (surbhi90) [inline stories/Molecules/Sidebar-data/sidebardata.scss:10]: @harishkumar-srijan Remove this commented code

File: stories/Molecules/Sidebar-data/sidebardata.scss [hunk 3]
```diff
@@ -24,14 +25,15 @@
 
   &--description {
     a {
+      @extend %img_none;
+
       border: none;
       display: block;
-      font-family: $Font-Family-ProximaRegular;
       font-size: $font-size-14;
+      // font-family: $Font-Family-ProximaNova;
```
Comment (surbhi90) [inline stories/Molecules/Sidebar-data/sidebardata.scss:33]: @harishkumar-srijan Remove this commented code

File: stories/Molecules/Text/Faq/Faq.scss [hunk 4]
```diff
@@ -10,45 +10,52 @@
...
+  overflow: hidden;
+  padding: $spacing-16 0 $spacing-24;
+  position: relative;
+
   &::before {
-    content: "";
+    @include transition(0.1s ease);
+
     background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
+    border-top-color: #f3f3f3;
```
Comment (surbhi90) [inline stories/Molecules/Text/Faq/Faq.scss:23]: @harishkumar-srijan If we have the color variable for this then please use that

File: stories/Organism/Blocks/FeaturedContentCard/FeaturedCard/featuredcard.scss [hunk 5]
```diff
@@ -4,27 +4,30 @@
...
-    &--headertext {
+    &-title {
       display: block;
-      margin-bottom: 36px;
+      margin-bottom: 2.25rem;
+
       @include devicebreak(medium) {
-        margin-top: 32px;
         margin-bottom: 0;
+        margin-top: 32px;
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/FeaturedCard/featuredcard.scss:13]: @harishkumar-srijan use rem for margin

File: stories/Organism/Sidebar/sidebar.scss [hunk 6]
```diff
@@ -63,7 +63,8 @@
     &__menu {
       border-bottom: none;
       display: block;
-      font-family: $Font-Family-ProximaBold;
+      font-family: $Font-Family-ProximaNova;
```
Comment (surbhi90) [inline stories/Organism/Sidebar/sidebar.scss:66]: @harishkumar-srijan Do we need the font family here

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Atom/Cards/Publication-thumbnail/publication-thumbnail.scss [hunk 7]
```diff
@@ -1,40 +1,45 @@
...
   z-index: 1;
-  background: linear-gradient( 180deg,#c9d0d6 70%, #c9d0d6 0, #414648 70.5%, #9ea5ac 0, #9ea5ac);
+
   &::before {
-    content: "";
-    left: 32px;
-    z-index: -2;
-    right: 20px;
-    height: 27px;
+    background: #7b8085;
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Cards/Publication-thumbnail/publication-thumbnail.scss:12]: We dont' have this variable and this color we never used again in the story.

File: stories/Atom/Cards/Publication-thumbnail/publication-thumbnail.scss [hunk 8]
```diff
@@ -1,40 +1,45 @@
-@import "../../../assets/scss/mixins";
+@import '../../../assets/scss/mixins';
 
 /* publication_thumbnail */
 .publication_thumbnail__image {
-  padding: 35px 47px;
+  background: linear-gradient(180deg, #c9d0d6 70%, #c9d0d6 0, #414648 70.5%, #9ea5ac 0, #9ea5ac);
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Cards/Publication-thumbnail/publication-thumbnail.scss:5]: We dont' have this variable and this color we never used again in the story.

File: stories/Molecules/Text/Faq/Faq.scss (hunk 4 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Text/Faq/Faq.scss:23]: We dont' have this variable and this color we never used again in the story.

File: stories/Atom/Global/Icons/icons.scss [hunk 9]
```diff
@@ -1,41 +1,47 @@
...
-  background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg) no-repeat center center  ;
+  height: 16px;
   transform: translateX(0);
   width: 20px;
+
   &:hover {
     background: url(#{$img-path-Icon}/arrow-left-expand-hover.svg) no-repeat center center;
     transform: translateX(10px);
-    transition: transform 0.4s cubic-bezier(0.4, 0.22, 0.28, 1); 
+    transition: transform 0.4s cubic-bezier(0.4, 0.22, 0.28, 1);
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Icons/icons.scss]: Done

File: stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.jsx [hunk 10]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './ProgressBarNavigation.scss';
 
 export const ProgressBarNavigation = ({ }) => (
-  <div className="progress-container">
-    <div className="progress-bar" />
+  <div className="progress__container">
+    <div className="progress__bar" />
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.jsx]: Done

File: stories/Atom/Reach-element/Table/table.scss [hunk 11]
```diff
@@ -69,17 +78,18 @@ table {
...
+  padding: $spacing-12 $spacing-40 $spacing-12 $spacing-12;
+  vertical-align: top;
 }
 
 thead {
   tr {
     th {
-      font-family: $Font-Family-ProximaSemiBold;
       background-color: $color-gray-300;
+      font-family: $Font-Family-ProximaNova;
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Reach-element/Table/table.scss]: Done

File: stories/Components/StatsSlider/statsslider.scss [hunk 12]
```diff
@@ -49,58 +55,66 @@
...
+      border: none;
       border-radius: 0;
+      cursor: pointer;
+      flex-grow: 1;
       height: 4px;
-      border: none;
-      background: transparent;
       position: relative;
+      transition: all 0.8s ease-in-out;
+      transition: all 0.8s ease-in-out;
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Components/StatsSlider/statsslider.scss]: Done

File: stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss [hunk 13]
```diff
@@ -1,118 +1,134 @@
...
+    @include devicebreak(small) {
+      box-direction: normal;
+      box-orient: horizontal;
+      display: box;
+      display: flex;
+      display: flexbox;
+      display: flex;
+      flex-flow: row wrap;
+      flex-flow: row wrap;
+      flex-flow: row wrap;
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss]: Done

File: stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss [hunk 14]
```diff
@@ -1,118 +1,134 @@
...
+
   &::before {
-    content: "";
+    content: '';
   }
+
   &:hover {
-    .feature__card_slide {
+    .feature__card-slide {
       transform: translateX(0);
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss]: Done

File: stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss [hunk 15]
```diff
@@ -1,118 +1,134 @@
...
-      display: block;
+      @include transform(translateX(-105%));
+
       background-color: $color-yellow;
+      display: block;
       height: 100%;
       position: absolute;
-      @include transform(translateX(-105%));
-      transition: transform 0.3s ;
+      transition: transform 0.3s;
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Molecules/Blocks/FeaturedContentCard/featuredContentCard.scss]: Done

File: stories/Molecules/Form/signup.scss (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Form/signup.scss]: Done

File: stories/Molecules/Sidebar-data/sidebardata.scss (hunk 2 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Sidebar-data/sidebardata.scss]: Done

File: stories/Molecules/Sidebar-data/sidebardata.scss (hunk 3 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Sidebar-data/sidebardata.scss]: Done

File: stories/Organism/Blocks/FeaturedContentCard/FeaturedCard/featuredcard.scss (hunk 5 above)
Comment (harishkumar-srijan) [reply stories/Organism/Blocks/FeaturedContentCard/FeaturedCard/featuredcard.scss]: Done

File: stories/Organism/Sidebar/sidebar.scss (hunk 6 above)
Comment (harishkumar-srijan) [reply stories/Organism/Sidebar/sidebar.scss]: Done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**